operator-wg/whitepaper/03_Capabilities.md

```diff
@@ -20,20 +20,18 @@ An operator should monitor the update, and rollback if there was a problem durin
 An operator should report the version of the resources and their health status during the process. If there was an error, the version reported should be the version that is actually been used.
 
 #### Backup
-**(Current) Issue: https://github.com/cncf/sig-app-delivery/issues/49**
 
-An operator’s capability to back up the application state and it’s data should give the user the certainty that he is able to restore from a failure if data is lost or compromised.
+This capability is for operators that manage data and ensure that the operator is able to create consistent backups. This backup should be done in a way that the user of the operator can be certain that it can be restored if data is lost or compromised. Furthermore, the status information provided, should give insights when the backup ran the last time and where it is located.
```

Comment:
```suggestion
This capability is for operators that manage data and ensure that the operator is able to create consistent backups. This backup should be done in a way that the user of the operator can be certain that it can be restored if data is lost or compromised. Furthermore, the status information provided should give insights about when the backup last ran and where it is located.
```